core/templates/pages/exploration-editor-page/history-tab/history-tab.component.html

```diff
@@ -407,6 +407,15 @@ <h3>
 
     .filter-container {
       margin: 0 0 20px;
+      width: 100%;
+    }
+
+    .filter-container input {
```

Comment:
Here and below, try to use (or define) class selectors for styling a component.

Reply:
Done, I have used class selectors for input tag and i think there is no need to use CSS selector for icon.

Reply:
It's important to make it specific. Think about a scenario where a new icon within the same container is introduced -- we may not want the same styling applied to that icon. So please define a class for the icon.

Reply:
> It's important to make it specific. Think about a scenario where a new icon within the same container is introduced -- we may not want the same styling applied to that icon. So please define a class for the icon.

Understood
Done, kindly check